packages/grafana-ui/src/components/SelectNG/types.ts

```diff
@@ -0,0 +1,23 @@
+import { SelectableValue } from '@grafana/data';
+import { Placement } from '@popperjs/core';
+
+export interface SelectNGProps<T> {
```

Comment:
I think it'd be a good idea to extend these props from whatever HTML element is the main one for the Select and then enable passing those props without explicitly defining them. This would enable passing `aria-label` prop, etc.